cmd/vulcan-zap/main.go

```diff
@@ -149,6 +154,39 @@ func main() {
 		logger.Debug("Waiting for spider results...")
 		time.Sleep(5 * time.Second)
 
+		logger.Debugf("Running AJAX spider %v levels deep...", opt.Depth)
+
+		client.AjaxSpider().SetOptionMaxCrawlDepth(opt.Depth)
+		resp, err = client.AjaxSpider().Scan(targetURL.String(), "", "", "")
+		if err != nil {
+			return fmt.Errorf("error executing the AJAX spider: %v", err)
```

Comment:
Same here I would use this, event if it is not a big deal: fmt.Errorf("error executing the AJAX spider: %w", err)

Reply:
Already addressed this one in the last commit :grin: